app.json

```diff
@@ -74,7 +74,9 @@
           "initialOrientation": "PORTRAIT"
         }
       ],
-      "sentry-expo"
+      "sentry-expo",
+      "./config-plugins/withAndroidMailQueries",
```

Comment:
this does not match the file name! so how could it work? please rename the file accordingly or remove it, if not necessary.

Reply:
Yes, you're right. I will update.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/001f367f34ae03dade0fa83279b273fd37231cee